cime_config/config_pes.xml

```diff
@@ -40,6 +40,44 @@
     </mach>
   </grid>
 
+  <grid name="a%ne30np4.pg3.+l%ne30np4.pg3.+oi%tnx1v4" >
+    <mach name="any">
+      <pes pesize="M" compset="any">
+              <comment>none</comment>
+              <ntasks>
+                <ntasks_atm>-15</ntasks_atm>
+                <ntasks_lnd>-9</ntasks_lnd>
+                <ntasks_rof>-15</ntasks_rof>
+                <ntasks_ice>-6</ntasks_ice>
+                <ntasks_ocn>-2</ntasks_ocn>
+                <ntasks_glc>-15</ntasks_glc>
+                <ntasks_wav>-4</ntasks_wav>
+                <ntasks_cpl>-15</ntasks_cpl>
+              </ntasks>
+              <nthrds>
+                <nthrds_atm>1</nthrds_atm>
+                <nthrds_lnd>1</nthrds_lnd>
+                <nthrds_rof>1</nthrds_rof>
+                <nthrds_ice>1</nthrds_ice>
+                <nthrds_ocn>1</nthrds_ocn>
+                <nthrds_glc>1</nthrds_glc>
+                <nthrds_wav>1</nthrds_wav>
+                <nthrds_cpl>1</nthrds_cpl>
+              </nthrds>
+              <rootpe>
+                <rootpe_atm>0</rootpe_atm>
+                <rootpe_lnd>-6</rootpe_lnd>
+                <rootpe_rof>0</rootpe_rof>
+                <rootpe_ice>0</rootpe_ice>
+                <rootpe_ocn>-15</rootpe_ocn>
+                <rootpe_glc>0</rootpe_glc>
+                <rootpe_wav>0</rootpe_wav>
+                <rootpe_cpl>0</rootpe_cpl>
+              </rootpe>
+      </pes>
+    </mach>
+  </grid>
+
```

Comment:
Can you move this block below the following one, so that we get the same stucture as elsewhere in the file?